### ISSUE TITLE
Duplicated section titles in the taglist pane

### ISSUE THREAD

**Issue by paulrougieux** [vim-latex/vim-latex#178]
The plugin [yegappan/taglist](https://github.com/yegappan/taglist) and the plugin [vim-latex](https://github.com/vim-latex/vim-latex) are very nice to navigate within the table of content of a latex document. There is an issue however on my system section titles are duplicated. Example based on [minimal latex](https://gist.github.com/Michael0x2a/e46e12a66b7dc604db5e).

![minimal_tex](https://user-images.githubusercontent.com/5647397/83230957-f79c5d00-a18a-11ea-93c4-576087cf5003.png)

I installed and updated both packages with Vundle recently.

How to prevent duplicated section titles in the taglist pane?

**Comment by gerw** [comment]
I don't think that this is related to vim-latex. Which executable do you use to generate your ctags information? Does this also happens if you do not load / uninstall vim-latex?

**Comment by paulrougieux** [comment]
If I disable vim-latex, the taglist appears without repetitions. But I still want vim-latex for citation completion and also the tag list is deeper with vim latex, because it also has figures and tables references.

**Comment by gerw** [comment]
In ok, it seems that vim-latex has some builtin taglist-support. In `ftplugin/latex-suite/texrc`, we find
````vim
" This option turns on definition of tags for LaTeX.
" NOTE: In taglist window you may see duplicate entries for the same tag. It
"       means some definitions are repeated somewhere else. You can turn off
"       internal definitions or remove external (read ctags description
"       where).
TexLet g:Tex_InternalTagsDefinitions = 1 
````
Hence, you should either set this variable to 0 or you have to check where some definitions are generated externally.

**Comment by paulrougieux** [comment]
Thanks [the explanation from texrc line 765](https://github.com/vim-latex/vim-latex/blob/dc2053cd45caeb4e61d461a7d889cee5d55b67d9/ftplugin/latex-suite/texrc#L765) seems to be the issue. I'm trying to figure out how to deactivate these definitions generated externally?
